csm/cli/support_bundle/support_bundle.py

```diff
@@ -52,57 +52,26 @@ def import_provisioner_plugin():
             return None
         return provisioner
 
-    @staticmethod
-    async def fetch_host_from_salt():
-        """
-        This method fetch hostname for all nodes from Salt DB
-        :return: hostnames : List of Hostname :type: List
-        :return: node_list : : List of Node Name :type: List
-        """
-        process = SimpleProcess(
-            "salt-call pillar.get cluster:node_list --out=json")
-        _out, _err, _rc = process.run()
-        if _rc != 0:
-            Log.warn(f"Salt command failed : {_err}")
-            return None, None
-        output = JsonMessage(_out.strip()).load()
-        nodes = output.get("local", [])
-        hostnames = []
-        for each_node in nodes:
-            process = SimpleProcess(
-                f"salt-call pillar.get cluster:{each_node}:hostname --out=json")
-            _out, _err, _rc = process.run()
-            if _rc != 0:
-                Log.warn(f"Salt command failed : {_err}")
-                return None, None
-            output = JsonMessage(_out.strip()).load()
-            hostnames.append(output.get("local", ""))
-        return hostnames, nodes
-
     @staticmethod
     async def fetch_host_from_cluster():
```

Comment:
fetch_host_from_cluster()---> fetch_active_nodes_hosts(), since cluster.sls is not in use

Reply:
done @pawankumarsrivastava